apps/web/src/components/Shared/Markup/MarkupLink/ExternalLink.tsx

```diff
@@ -9,7 +9,7 @@ const ExternalLink: FC<MarkupLinkProps> = ({ href, title = href }) => {
   }
 
   let link = href;
-  if (!href.includes('://')) {
+  if (!link.startsWith('https://')) {
```

Comment:
how about just http? tbh this should be solved by regex, string checks will fail on some criteria, ex: user can just use https:// to explain what https is in his post